### PR TITLE
[DO NOT MERGE] add IE swagger specs

### PR DESCRIPTION
**Pull request by ferno-ms**
This change adds two parameters, "sqlServerResourceId" and "storageAccountResourceId" to the database Import/Export operations, and to the server Import operation. These parameters are used to achieve network isolation for these requests.

### Contribution checklist:
- [x] I have reviewed the [documentation](https://aka.ms/ameonboard) for the workflow.
- [x] [Validation tools](https://aka.ms/swaggertools) were run on swagger spec(s) and have all been fixed in this PR. [How to fix?](https://aka.ms/ci-fix)

If any further question about AME onboarding or validation tools, please view the [FAQ](https://aka.ms/faqinprreview).

### ARM API Review Checklist
- [x] Service team MUST add the "**WaitForARMFeedback**" label if the management plane API changes fall into one of the below categories. 
  - adding/removing APIs.
  - adding/removing properties.
  - adding/removing API-version. 
  - adding a new service in Azure.

<i>Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.</i>

- [x] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://aka.ms/SwaggerPRReview).

### CODE REVIEW

**Comment by ferno-ms** [comment]
Someone should add the WaitForARMFeedback label to this, I do not have permission.

**Comment by erich-wang** [comment]
Added label "WaitForARMFeedback" to request review from ARM review board

File: specification/sql/resource-manager/Microsoft.Sql/preview/2020-02-02-preview/importExport.json [hunk 1]
```diff
@@ -0,0 +1,552 @@
...
+          ]
+        },
+        "administratorLogin": {
+          "description": "Administrator login name.",
+          "type": "string",
+          "x-ms-mutability": [
+            "create"
+          ]
+        },
+        "administratorLoginPassword": {
```
Comment (chiragg4u) [inline specification/sql/resource-manager/Microsoft.Sql/preview/2020-02-02-preview/importExport.json:267]: >administratorLoginPassword [](start = 9, length = 26)

This as well as storage key please use secret annotation. [https://armwiki.azurewebsites.net/rp_onboarding/process/property_design_best_practices.html#secrets](https://armwiki.azurewebsites.net/rp_onboarding/process/property_design_best_practices.html#secrets)

File: specification/sql/resource-manager/Microsoft.Sql/preview/2020-02-02-preview/importExport.json [hunk 2]
```diff
@@ -0,0 +1,552 @@
...
+          "description": "Administrator login password.",
+          "type": "string",
```
Comment (chiragg4u) [inline specification/sql/resource-manager/Microsoft.Sql/preview/2020-02-02-preview/importExport.json:269]: >         "type": "string", [](start = 1, length = 26)

Isn't passing the server/DB password as string a sequrity risk? Many components in the control path may put these in their logs (may be in error paths).

**Comment by chiragg4u** [comment]
Added minor comments, few questions
1. There is no server level export function, wouldn't this be required?
2. Is server import action, just imports the database? 
3. Would this be similar to MOVE if done within a cloud?
4. What will happen if multiple export calls are being called? will there be some kind of throttling?

**Comment by ravbhatnagar** [comment]
@ferno-ms - comments from previous ARM review have not been addressed.

**Comment by erich-wang** [comment]
@ferno-ms , could you please address the comments from ARM reviewer? And please resolve the conflict as well.

**Comment by erich-wang** [comment]
@ferno-ms, is this PR still valid? we'll close this PR if no response from your side by end of Aug 14, thanks.

**Comment by ferno-ms** [comment]
Deployment of the code to production has been delayed for a long time so we are still not able to check this in as the new parameters are not available in any production cluster. (I also plan to fix comments at that time.)
As I understand the changes were deployed to stage last week, so I expect them to be in production soon.

**Comment by erich-wang** [comment]
@ferno-ms, is this PR still valid? Please resolve the conflict if yes, thanks.

**Pull request closed without merging**